src/app/components/select-items/select-items.component.spec.ts

```diff
@@ -13,11 +14,38 @@ describe('SelectItemsComponent', () => {
       .compileComponents();
 
     fixture = TestBed.createComponent(SelectItemsComponent);
+    fixture.componentRef.setInput('possibleItemNames', possibleItemNames);
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should add empty item', () => {
+    component.addItem();
+    expect(component.getSelectedItem()).toBe('');
+  });
+
+  it('should display a list of all possible items', () => {
+    const items = fixture.nativeElement.querySelectorAll('option');
+    const numberOfDefaultItems = 1
+    expect(items.length).toBe(possibleItemNames.length + numberOfDefaultItems);
+  })
+
+  it('should get value from selected item', ()=> {
+    const selectedItem = 'item 1';
+    component.itemSubmitForm.get('selectedItem')?.setValue(selectedItem);
+    expect(component.getSelectedItem()).toBe(selectedItem);
```

Comment:
In this test too, we're actually performing the actions in the test and not relying on any code from the component's file (`select-items.component.ts`), when this happens, we can assume this test is perhaps not required as it doesn't test any unit in our case.